lib/galaxy/security/__init__.py

```diff
@@ -2,11 +2,24 @@
 Galaxy Security
 
 """
+from typing import (
+    List,
+    Optional,
+)
+
+from typing_extensions import Literal
+
 from galaxy.util.bunch import Bunch
 
+ActionModel = Literal["grant", "restrict"]
+
 
 class Action:
-    def __init__(self, action, description, model):
+    action: str
+    description: str
+    model: ActionModel
```

Comment:
Shouldn't mypy figure this out without the explicit hint ?

Reply:
I'm old school - I prefer properties are annotated and have types. Now just making this a dataclass so that constructor isn't needed at all seems like a better a path forward if the verboseness is problematic.